src/OVAL/probes/unix/process58_probe.c

```diff
@@ -108,15 +108,15 @@ extern char const *_cap_names[];
 
 /* Convenience structure for the results being reported */
 struct result_info {
-        const char *command_line;
-        const char *exec_time;
-        unsigned pid;
-        unsigned ppid;
-        long priority;
+	const char *command_line;
+	const char *exec_time;
+	unsigned pid;
+	unsigned ppid;
+	long priority;
 	int ruid;
-        const char *scheduling_class;
-        const char *start_time;
-        const char *tty;
+	const char *scheduling_class;
+	const char *start_time;
+	const char *tty;
```

Comment:
please don't do only formatting changes

Reply:
And the same question here. When we are going to fix it? Asking again because this formatting error is different (could be mitigated semi-automatically by a code beautifier).

Reply:
Probably never, only if we need eg. rename these variables or refactor the code.

We enforce the coding style only on new or edited code. This is because people often apply their patches on top of our code and they will have to rework them if you change the indents or code style.

Reply:
Reverted.